j-mail/src/handler.ts

```diff
@@ -78,8 +93,12 @@ const getDataForUpdateEmail = async (
           comment: note.comment,
           thread: note.comment.thread,
           post: note.comment.thread.post,
+          image: note.comment.thread.post.images.find((image) => image.imageRole === ImageRole.HEADLINE)?.smallSize || 'https://dlke4x4hpr6qb.cloudfront.net/sample-post-img.jpg',
+          commentAuthor: note.comment.author.handle,
         })
       }
+    } else if (note.type === NotificationType.THREAD_COMMENT_THANKS) {
+      thanksCount++
```

Comment:
This isn't the most critical thing in the world, but to handle un-thanking correctly we should check that the thanks record still exists (which I think translates to checking that `note.commentThanksId !== null`)